components/logins/ios/Logins/Extensions/String+Free_Logins.swift

```diff
@@ -4,9 +4,17 @@
 
 import Foundation
 
-extension String {
-    public init(freeingRustString rustString: UnsafeMutablePointer<CChar>) {
-        defer { sync15_passwords_destroy_string(rustString) }
-        self.init(cString: rustString)
-    }
-}
+/*
+ * NOTE: This is now provided by Glean.
+ * Because it's all compiled into a single libmegazord.a it doesn't matter which `destroy_string` we call,
+ * internally it defers to the same string implementation.
+ * Because Glean is added as a submodule it's easier to change this occurence than it it to change the one of Glean.
+ */
+/*
```

Comment:
I'm curious what the underlying conflict was here - was it swift complaining about trying to add two extension methods with the same name on the `String` class?

Reply:
Yes, we have the same extension to `String` with the same named parameter.
I actually propose to rename it on our side (`s/Rust/Glean/`) to avoid the conflict, but as I also want to submodule based on the release commit this has to be defered to the next release.